content/21.nutraceuticals.md

```diff
@@ -152,8 +240,14 @@ Some studies suggest that probiotics are beneficial against common viral infecti
 Probiotics and next-generation probiotics, which are more akin to pharmacological-grade supplements, have been associated with multiple potential beneficial effects for allergies, digestive tract disorders, and even metabolic diseases through their anti-inflammatory and immunomodulatory effects [@doi:10.1038/nmicrobiol.2017.57; @doi:10.1093/advances/nmy063].
 However, the mechanisms by which probiotics affect these various conditions would likely differ among strains, with the ultimate effect of the probiotic depending on the heterogeneous set of bacteria present [@doi:10.1093/advances/nmy063].
 Some of the beneficial effects of probiotics include reducing inflammation by promoting the expression of anti-inflammatory mediators, inhibiting toll-like receptors (TLR) 2 and 4, direct competition with pathogens, the synthesis of antimicrobial substances or other metabolites, improving intestinal barrier function, and/or favorably altering the gut microbiota and the brain-gut axis [@doi:10.1016/j.earlhumdev.2019.05.010; @doi:10.1093/advances/nmy063; @doi:10.1159/000342079].
+It is also thought that lactobacilli such as _Lactobacillus paracasei, Lactobacillus plantarum_ and _Lactobacillus rhamnosus_ have the capacity to bind and inactivate some viruses via adsoptive and/or trapping mechanisms [@doi:10.1016/j.ijfoodmicro.2006.10.044].
+Other probiotic lactobacilli and even non-viable bacterium-like particles have been shown to reduce viral attachment to host cells and reduce viral titers, reduce cytokine synthesis, enhance the antiviral IFN-α response along with inducing various other antiviral mechanisms  [@doi:10.1111/lam.12160; @doi:10.1016/j.anaerobe.2011.04.022; @doi:10.4049/jimmunol.1800927; @doi:10.1016/j.ijfoodmicro.2006.10.044; @doi:10.3390/medicina55090519; @doi:10.1038/s41598-019-39602-7; @doi:10.3920/bm2016.0160; @doi:10.3389%2Ffimmu.2020.00015; @doi:10.1016/j.intimp.2013.12.020; @doi:10.1016/j.intimp.2019.106115].
```

Comment:
```suggestion
Other probiotic lactobacilli and even non-viable bacterium-like particles have been shown to reduce both viral attachment to host cells and viral titers, along with reducing cytokine synthesis, enhancing the antiviral IFN-α response, and inducing various other antiviral mechanisms  [@doi:10.1111/lam.12160; @doi:10.1016/j.anaerobe.2011.04.022; @doi:10.4049/jimmunol.1800927; @doi:10.1016/j.ijfoodmicro.2006.10.044; @doi:10.3390/medicina55090519; @doi:10.1038/s41598-019-39602-7; @doi:10.3920/bm2016.0160; @doi:10.3389%2Ffimmu.2020.00015; @doi:10.1016/j.intimp.2013.12.020; @doi:10.1016/j.intimp.2019.106115].
```
Just an idea since there were a bunch of "reduce"s in a row!